packages/next-swc/Cargo.toml

```diff
@@ -46,38 +46,38 @@ swc_emotion = { version = "0.29.10" }
-turbo-malloc = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3", default-features = false }
-turbo-tasks = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-build = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-bytes = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-env = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-fetch = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3", default-features = false }
-turbo-tasks-fs = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-hash = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-macros = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-macros-shared = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-memory = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-tasks-testing = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbo-updater = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-cli-utils = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-core = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-create-test-app = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-css = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-dev = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-dev-server = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-ecmascript = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-env = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-json = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-mdx = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-node = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-static = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-swc-utils = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-test-utils = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
-turbopack-tests = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230323.3" }
+auto-hash-map = { git = "https://github.com/vercel/turbo.git", branch = "jrl-rsc-streaming" }
```

Comment:
Is something blocking https://github.com/vercel/turbo/pull/4329? Ideally we land that to a turbopack release before landing this here

Reply:
The opposite, we're blocked by https://github.com/vercel/turbo/pull/4329 which implements the `Stream<T>` struct. We'll always land breaking changes first in Turbopack, then update the Next.js PR quickly to the a tagged release, and merge this.